### PR TITLE
(#341) Regression test fixes

### LINKED ISSUE #341: Regression tests maintenance
*Description of what is to be accomplished.*
- there is an issue with alternating lists that have come up a few times ( which I think is due to different screen rendering from PC and Mac) 
- blog archive will not show 2016 in 2023.. so get rid of it 
- a bunch of analytics failures based on adobe launch rules changes
- header test had one step missing in a clean up scenario to select a default "show for all pages" radio button ( otherwise it will not render a header, and I know why it wasn't caught with a prev maintanance ticket, because I blamed an ODE config being messed up and not having a mega menu YAML for some sections, but since updating dev-ac with redesign, this is an existing issue that was missed 2 weeks ago )
**Parent Issue:** #9999

### Sub-Tasks
- [ ] Determine if this change will impact existing piece(s) of documentation (e.g., comps, instruction files, etc.). If so, create a new ticket for the associated updates to be completed after Drupal development (or final stage) is completed.
- [ ] Create Component
- [ ] Implement Unit Tests for Component

### Notes

*Some complementary notes if necessary:*
* > Here goes a quote from an email
* Here goes whatever useful information can exist…
* More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by kate-mashkina**
Closes #341

### CODE REVIEW

**Comment by kate-mashkina** [comment]
Screen Shot 2023-01-11 at 4 33 29 PM
One failure is coming from drug dictionary analytics( its on my to do list to refactor analytics capturing logic)

File: cypress/e2e/BlogSeriesBlogPosts.feature [hunk 1]
```diff
@@ -50,7 +50,6 @@ Feature: As an user, I want to see different components of the Blog Series and B
             | 2019  | false      |
             | 2018  | false      |
             | 2017  | false      |
-            | 2016  | false      |
```
Comment (blairlearn) [inline cypress/e2e/BlogSeriesBlogPosts.feature:53]: Along with the removal of years which are no longer available, should this also be expanded to include 2020 through 2023?

File: cypress/e2e/BlogSeriesBlogPosts.feature [hunk 2]
```diff
@@ -60,7 +59,6 @@ Feature: As an user, I want to see different components of the Blog Series and B
             | 2019  | true       |
             | 2018  | true       |
             | 2017  | false      |
-            | 2016  | false      |
```
Comment (blairlearn) [inline cypress/e2e/BlogSeriesBlogPosts.feature:63]: Same question

**Review by blairlearn — COMMENTED** [review]
Just a couple general questions, nothing alarming.

**Pull request merged**